imdg-clients.txt

```diff
@@ -1751,6 +1751,14 @@ ReleaseNotes:
 ---
 ========== Python Client
 ---
+Version: 5.4.0
+Date: 20/05/2024
+Download: https://files.pythonhosted.org/packages/84/82/8b7304ad9a7c43a7f6a0fa6af1f4d21994afdeec96a74c47307059382cf1/hazelcast_python_client-5.4.0.tar.gz
+Download_Size: 247.8 KB
+Github: https://github.com/hazelcast/hazelcast-python-client/tree/v5.4.0
+Docs: https://github.com/hazelcast/hazelcast-python-client/blob/v5.4.0/README.rst
+APIDocs: https://hazelcast.readthedocs.io/en/v5.4.0/
```

Comment:
doesn't work https://hazelcast.readthedocs.io/en/v5.4.0/

one for v5.3.0 works

Reply:
Seems like it's some weird feature of readthedocs. See this image:
 ![image](https://github.com/user-attachments/assets/19511381-acce-41b9-8ca1-6386944b5e9c)

@yuce wdyt?

Reply:
Alternatively, somehow v5.4.0 is not added to readthedocs during release

Reply:
> doesn't work https://hazelcast.readthedocs.io/en/v5.4.0/
> 
> one for v5.3.0 works

I would _guess_ we'll fix this so I think it's better to release with a broken link, than not to release at all.

Reply:
Found some instructions here https://hazelcast.atlassian.net/wiki/spaces/HZC/pages/115376462/Python+Client+Release+Procedure

Looks like GitHub webhook was missing secret for readthedocs.org so v5.4.0 has not been synced and built. I tried adding secret but failed to resync. May be it will resync by itself

@JackPGreen I totally agree with your sentiments but the broken API docs link appears in download page so prefer we investigate a little further as its unhelpful I think.  @srknzl who normally deals with this in the team?

Worst case we can leave value blank but keep the key. I think website can cope with that but not sure (for Platform we do [this](https://github.com/hazelcast/rel-scripts/blob/master/hazelcast-enterprise.txt#L17))

Lets see if anything can be done without spending too much time; otherwise, try blank value (merge PR and check website) or worst case leave it broken

Reply:
> Looks like GitHub webhook was missing secret for readthedocs.org so v5.4.0 has not been synced and built. I tried adding secret but failed to resync. May be it will resync by itself

I don't think it's related to GitHub actions or secrets.

> @JackPGreen I totally agree with your sentiments but the broken API docs link appears in download page so prefer we investigate a little further as its unhelpful I think. @srknzl who normally deals with this in the team?

FYI It's also under discussion [here](https://hazelcast.slack.com/archives/C016TGXPWPJ/p1722529119680379?thread_ts=1722525946.636629&cid=C016TGXPWPJ).

> Lets see if anything can be done without spending too much time

https://github.com/hazelcast/hazelcast-python-client/pull/689

Reply:
> https://github.com/hazelcast/hazelcast-python-client/pull/689

Ah yes yes forgot to mention the config file as well. The odd thing is this file is even missing in v5.3.0. May be its a new requirement

It was to do with the secret I think as I saw error in one of the build and now v5.4.0 has appeared likely as I added the secret in GH

But v5.4.0  fails as file is missing https://readthedocs.org/projects/hazelcast/builds/25189318/ because it needs to be part of the tag. 

![image](https://github.com/user-attachments/assets/5ce8ad0d-4ea9-47d6-8dc6-93df99ee62b7) 

We could [add](https://stackoverflow.com/a/50454106) the file to the tag but doesn't feel right. So I think we should keep it blank.

Reply:
Just a thought...

Another option could be to create another tag like `v5.4.0-doc` to include the file and once docs are built, use that as the link

Reply:
I’d totally forgotten that it’s a tagged version, good point - there’s nothing we can do retrospectively. Ill remove it instead.

Reply:
765db359ed0dedea775857bdc8719347cce86f04